server/routes/partners.js

```diff
@@ -0,0 +1,28 @@
+const validPartners = {
+	labs : 'labs.ft.com'
```

Comment:
Ideally these would not point to the organisation's site but instead point to their ft.com registration/login page.